test/parallel/test-fs-whatwg-url.js

```diff
@@ -96,8 +96,8 @@ if (common.isWindows) {
     {
       code: 'ERR_INVALID_ARG_VALUE',
       name: 'TypeError',
-      message: "The argument 'path' must be a string or Uint8Array without " +
-               "null bytes. Received '/tmp/\\x00test'"
```

Comment:
I suggest to keep the double quotes around.

Reply:
Yep I agree ...